packages/stacking/README.md

```diff
@@ -145,3 +160,95 @@ stackingStatus:
 //   }
 // }
 ```
+
+## Delegation
+
+There are four methods available for delegation, two for the delegators and two for the delegatee.
+
+### Delegatee
+
+If you are the account owner / stacker / delegatee, you can delegate or revoke delegation rights.
+
+#### Delegate STX
+
+```typescript
+// STX address of the delegator
+const delegateTo = 'ST2MCYPWTFMD2MGR5YY695EJG0G1R4J2BTJPRGM7H';
+// burn height at which the delegation relationship should be revoked (optional)
+const untilBurnBlockHeight = 5000;
+
+const delegetateResponse = await client.delegateStx({
+  amountMicroStx,
+  delegateTo,
+  untilBurnBlockHeight, // optional
+  poxAddress, // optional
+  privateKey,
+});
+
+// {
+//   txid: '0xf6e9dbf6a26c1b73a14738606cb2232375d1b440246e6bbc14a45b3a66618481',
+// }
+```
+
+#### Revoke delegation
+
+```typescript
+// note that the parameter here is not JSON
+const revokeResponse = await client.revokeDelegateStx(privateKey);
+
+// {
+//   txid: '0xf6e9dbf6a26c1b73a14738606cb2232375d1b440246e6bbc14a45b3a66618481',
+// }
+```
+
+### Delegator
+
+If you are the delegator, you can stack / lock up tokens for your delegatees and commit to stacking participation for upcoming reward cycles.
+
+#### Stack delegated STX
+
+```typescript
+// delegators would initiate a different client
+const delegatorAddress = 'ST22X605P0QX2BJC3NXEENXDPFCNJPHE02DTX5V74';
+// delegator private key for transaction signing
+const delegatorPrivateKey = 'd48f215481c16cbe6426f8e557df9b78895661971d71735126545abddcd5377001';
+// the BTC address for reward payouts; either to the delegator or to the BTC address set by the account holder
+const delegatePoxAddress = 'msiYwJCvXEzjgq6hDwD9ueBka6MTfN962Z';
```

Comment:
We need to update the comment here that this is only the delegator's BTC address. We should also include that this is optional. I would love to see some link or comment about how this works, since it's been prone to confusion. IMO, this should be omitted by default. If we do keep it, the variable should be renamed to `delegatorPoxAddress`

Reply:
I'll remove it for now

Reply:
btw, it is mentioned to be optional where the user initiates the delegation relationship